src/components/DataTable/index.jsx

```diff
@@ -102,6 +103,7 @@ class DataTable extends Component {
         {title && (
           <Toolbar className={classes.toolbar}>
```

Comment:
We should probably take out the `Toolbar` component outside and let `DataTable` only take care of rendering a table. Are we able to do this?

```jsx
{
 displayTableLayout ?
   <DataTable ...> :
   <DataCard ...>
}
```

Reply:
We should also have the table icon be beside the filter icon instead of being by itself in the center:

![screen shot 2018-07-05 at 9 50 58 am](https://user-images.githubusercontent.com/3766511/42327148-17291674-8039-11e8-87d9-a0cea4dd68c5.png)